modules/ROOT/pages/clustering/setup/encryption.adoc

```diff
@@ -65,9 +65,9 @@ This means that the server now has the capability of establishing trust with oth
 
 [CAUTION]
 ====
-Be sure to exercise caution when using CA certificates in the `trusted` directory, as any certificates signed by that CA are then trusted to join the cluster.
```

Comment:
I think it's worth keeping the fragment `any certificates signed by that CA are trusted to join the cluster`, because it explains the reason for this guidance, and the consequences of not following it.

What was the reason for removing it?

Reply:
I thought this explanation provided some kind of clue about the possible malicious use of certificates signed by CA in the `trusted` directory. That's why I removed it.

Reply:
But if this removal makes our instructions less clear, I'll undo it.

Reply:
I guess it does say
> any certificate signed by the CA is thus trusted

a couple of lines above, but outside the caution box.

Reply:
I'd say no harm to reiterate the message as "caution".